src/config.ts

```diff
@@ -429,9 +442,11 @@ export const GAS_SCHEDULE_V1: FeeSchedule = {
         const firstHop = (fillData as MultiHopFillData).firstHopSource;
         const secondHop = (fillData as MultiHopFillData).secondHopSource;
         const firstHopGas =
-            GAS_SCHEDULE_V1[firstHop.source] === undefined ? 0 : GAS_SCHEDULE_V1[firstHop.source](firstHop.fillData);
+            GAS_SCHEDULE_V1[firstHop.source] === undefined ? 0 : GAS_SCHEDULE_V1[firstHop.source]!(firstHop.fillData);
```

Comment:
is this because it doesn't think `firstHop.source` is an `ERC20BridgeSource`? Or because the type of `GAS_SCHEDULE_V1` doesn't explicitly say it contains a values for all `ERC20BridgeSource`.

Reply:
It seems like TS is not able to infer that `GAS_SCHEDULE_V1[firstHop.source]` will always be defined from the preceding ternary. I will update to a more TS friendly expression 👍